ui/handle_input.go

```diff
@@ -236,42 +276,99 @@ func (win *Window) HandleInputs() {
 		}
 	}
 
-	for win.inputs.receiveIcons.copy.Clicked(win.gtx) {
+	if win.inputs.receiveIcons.copy.Clicked(win.gtx) {
 		clipboard.WriteAll(win.walletInfo.Wallets[win.selected].Accounts[win.selectedAccount].CurrentAddress)
 		win.addressCopiedLabel.Text = "Address Copied"
 		time.AfterFunc(time.Second*3, func() {
 			win.addressCopiedLabel.Text = ""
 		})
 		return
 	}
+
+	if strings.Trim(win.inputs.spendingPassword.Text(), " ") != "" {
+		strength := (dcrlibwallet.ShannonEntropy(win.inputs.spendingPassword.Text()) / 4.0)
```

Comment:
parentheses around "dcrlibwallet.ShannonEntropy(win.inputs.spendingPassword.Text()) / 4.0" is redundant.